packages/odie-client/src/types/index.ts

```diff
@@ -53,7 +54,7 @@ export type Context = {
 	site_id: number | null;
 	user_tracking?: OdieUserTracking[];
 	sources?: Source[];
-	prompt_tags?: {
```

Comment:
Is this change unrelated? Or was this a bug?

Reply:
I changed it because I inspected the return type of Message and saw this!

<img width="1041" alt="image" src="https://github.com/user-attachments/assets/b3373655-dd4f-44fc-b664-c17c0308be05">